scene/gui/tree.cpp

```diff
@@ -2214,6 +2214,26 @@ void Tree::_gui_input(InputEvent p_event) {
 
 				if (b.button_index == BUTTON_LEFT) {
 
+					Ref<StyleBox> bg = cache.bg;
+
+					Point2 pos = Point2(b.x, b.y) - bg->get_offset();
```

Comment:
The assignments of `bg` and `pos` are only relevant if the following condition is met, so they should be in the `if` block.